index.html

```diff
@@ -113,10 +118,15 @@ <h2 style="text-align:center;">Runs on Any of These Kubernetes Environments</h2>
         <h2 style="text-align:center;">Get Up and Running in 5 Minutes</h2>
 
         <div class="getting-started">
-
-            <div class="step-box">1. Schedule a demo</div>
-            <div class="step-box">2. Config your Costs</div>
-            <div class="step-box">3. Start Optimizing!</div>
+            <a href="https://github.com/kruize/autotune/blob/master/docs/autotune_install.md">
+                <div class="step-box">1. Setup Kruize</div>
+            </a>
+            <a href="https://kruize.github.io/kruize-website/docs/performance_profiles.html">
+                <div class="step-box">2. Configure your Costs</div>
+            </a>
+            <a href="https://kruize.github.io/kruize-website/docs/lm_create_exp.html">
+                <div class="step-box">3. Start Optimizing!</div>
```

Comment:
Same here as well, since we cant be sure that the user is looking into the local use case I dont think we should link the local monitoring create exp link.